ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -53,7 +53,7 @@ import { Body } from "../../components/core/typography/Body";
 import { CreditCard } from "../../types/pagopa";
 import { BlockButtonProps } from "../../components/ui/BlockButtons";
 
-type NavigationParams = Readonly<{
+export type NavigationParams = Readonly<{
```

Comment:
To keep code encapsulation strong, we should avoid to expose types, functions, const, whatever only for tests purposes: this type, when exported, could be used by another developer for a new code. In this way we allow a not wanted dependency

Instead you could use this approach (or a better one if you know)
https://github.com/pagopa/io-app/blob/ff4b0f6eec44d341c9f611bb0239b83fab7fa199/ts/sagas/startup/watchCheckSessionSaga.ts#L94
and the make a not null assert inside the test code
`testableCheckSession!`

Reply:
You're right! 
Instead of a ternary if condition, solved as suggested with:

```typescript
  type NavigationParams = InferNavigationParams<typeof AddCardScreen>;

  const params: NavigationParams = {
    inPayment: none
  } as NavigationParams;
```

Solved in [1961fa8](https://github.com/pagopa/io-app/pull/2833/commits/1961fa86b287a24fcc5fd1436f009140bbac6af7)